pages/components/DataTable.neu.page.ts

```diff
@@ -0,0 +1,85 @@
+import { expect, type Locator, Page } from '@playwright/test';
+
+export class DataTable {
+    /* since the table is within Vuetify's jurisdiction,
+        we cannot specify test ids for Playwright and heavily rely on classes as locators */
+    readonly page: Page;
+    readonly tableLocator: Locator;
+
+    constructor(page: Page, locator: Locator) {
+        this.page = page;
+        this.tableLocator = locator;
+    }
+
+    /* actions */
+    public getItemByText(expectedText: string): Locator {
+      return this.tableLocator.getByRole('cell', { name: expectedText, exact: true });
+    }
+
+    public setItemsPerPage(value: string): void {
+      const itemsPerPageSelect: Locator = this.page.locator('.v-data-table-footer__items-per-page .v-select');
+
+      itemsPerPageSelect.selectOption(value);
+    }
+
+    public async goToFirstPage(): Promise<void> {
+      const goToFirstPageButton: Locator = this.page.locator('.v-pagination__first button:not(.v-btn--disabled)');
+      
+      await goToFirstPageButton.click();
+    }
+
+    public async goToPreviousPage(): Promise<void> {
+      const goToPreviousPageButton: Locator = this.page.locator('.v-pagination__prev button:not(.v-btn--disabled)');
+      
+      await goToPreviousPageButton.click();
+    }
+
+    public async goToNextPage(): Promise<void> {
+      const goToNextPageButton: Locator = this.page.locator('.v-pagination__next button:not(.v-btn--disabled)');
+      
+      await goToNextPageButton.click();
+    }
+
+    public async goToLastPage(): Promise<void> {
+      const goToLastPageButton: Locator = this.page.locator('.v-pagination__last button:not(.v-btn--disabled)');
+      
+      await goToLastPageButton.click();
+    }
+
+    /* assertions */
+    public async checkCurrentPageNumber(expectedPageNumber: number): Promise<void> {
+      const currentPageNumberElement: Locator = this.page.locator('.v-pagination__item');
+      const currentPageNumberText: string | null = await currentPageNumberElement.textContent();
+
+      await expect(currentPageNumberText).toEqual(expectedPageNumber);
```

Comment:
Here a number is compared to a string. I think you could do:

```suggestion
      await expect(Number(currentPageNumberText)).toBe(expectedPageNumber);
```